### PR TITLE
Post launch feedback updates

### PR DESCRIPTION
**Pull request by kattylucy**
- Add rating to pool card (landing page)
- Cleanup types a bit and unused variables

#2505 

- [ ] Dev

### CODE REVIEW

**Review by sophialittlejohn — COMMENTED** [review]
Looks like there may be a padding missing between the Rating chips and the border around the card?

Screenshot 2024-10-21 at 09 31 18

**Review by hieronx — COMMENTED** [review]
Reviewed all the UI fixes after the feedback I left in the ticket and everything looks perfect to me, great work!

File: centrifuge-app/src/components/PoolOverview/KeyMetrics.tsx [hunk 1]
```diff
@@ -46,6 +46,13 @@ type Tranche = Pick<DailyTrancheState, 'id'> & {
 
 type TinlakeDataKey = keyof typeof tinlakeData
 
+export type RatingType = {
+  agency?: string
+  reportUrl?: string
+  reportFile?: any
+  value?: string
+}
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/PoolOverview/KeyMetrics.tsx:54]: would extend this type from the metadata instead of creating a new type

**Review by sophialittlejohn — APPROVED** [review]
Just a few type things but looks really good otherwise!

**Pull request merged**